jekyll/_cci2/docker-layer-caching.md

```diff
@@ -117,56 +117,95 @@ Let's use the following Dockerfile to illustrate how Docker Layer Caching works.
 {:.no_toc}
 
 ```
-FROM elixir:1.6.5
+FROM elixir:1.11.4
 
-# Make apt non-interactive
+# make Apt non-interactive
 RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90circleci \
   && echo 'DPkg::Options "--force-confnew";' >> /etc/apt/apt.conf.d/90circleci
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+# Debian Jessie is EOL'd and original repos don't work.
```

Comment:
```suggestion
# Debian Jessie is EOL'd and original repos don not work.
```

Reply:
I'll edit it and remove the extra `n` @thomasr8